modules/apps/transfer/keeper/relay.go

```diff
@@ -572,3 +493,20 @@ func createPacketDataBytesFromVersion(appVersion, sender, receiver, memo string,
 
 	return packetDataBytes
 }
+
+// This function sends coins from the account to the transfer module account and then burn them.
+// We do this because bankKeeper.BurnCoins only works with a module account in SDK v0.50,
+// the next version of the SDK will allow burning coins from any account.
+// TODO: remove this function once we switch forwarding address to a module account (#6561)
+func (k Keeper) burnCoin(ctx sdk.Context, account sdk.AccAddress, coin sdk.Coin) error {
```

Comment:
another godoc nit, not sure if linter complains about unexported symbols godocs, but normally for exported symbols it says "should start with func name" - maybe we should do it here also:

e.g. `burnCoins sends coins from the account to the transfer module account ...`

Reply:
done